Cargo.toml

```diff
@@ -87,6 +88,7 @@ oak_channel = { path = "./oak_channel" }
 oak_client = { path = "./oak_client" }
 oak_core = { path = "./oak_core" }
 oak_crypto = { path = "./oak_crypto" }
+oak_containers_orchestrator_ipc_server = { path = "./oak_containers_orchestrator_ipc_server" }
```

Comment:
I think we already have too many crates. Should we just put this server inside the Orchestrator? (since it's not going to be reused anywhere else)

Reply:
We could, but I personally like having small crates, since it makes it easier to see which dependencies are used for what. :) 

Do we have any docs or norms about when we prefer to use seperate vs monolithic crates? I think it might be nice to have.

Reply:
This crate will never be used for anything other than that binary. It is both necessary and sufficient, so it should just be a mod IMO.

Reply:
alright, merged into one crate